app/views/escorts/_risk.html.slim

```diff
@@ -1,12 +1,12 @@
 #risk
   = render partial: 'status',
-    locals: { workflow: risk,
-              not_started_path: new_escort_risks_path(escort),
-              summary_path: escort_risks_path(escort) }
+    locals: { workflow: risk, not_started_path: new_escort_risks_path(escort), summary_path: escort_risks_path(escort) }
 
   h2
     ' Risk
-    / small 16:44 | 04 Apr 2016
+    - if risk&.reviewer
+      small
+        = t('escort.confirmed_timestamp', name: risk.reviewer.full_name, time: risk.reviewed_at)
   - if risk
```

Comment:
Also, the new changes were added outside this condition, but seems more logical to be inside it.